lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml

```diff
@@ -2558,3 +2558,89 @@ definitions:
         pattern: '^(\d{2,3})$'
         example: '01'
         x-nullable: false
+
+  network_probe_task:
```

Comment:
These network probe definitions are defined both in this yaml and the nprobe swagger yaml. It doesn't look like they are used in this file so just delete these.